.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+<!--
+Thank you for your interest in and contributing to Synapse Operator!
+Please, provide some information about your PR before proceeding.
+-->
+
+### Overview
+
+<!-- A high level overview of the change -->
+
+### Rationale
+
+<!-- The reason the change is needed -->
+
+### Juju Events Changes
+
+<!-- Any changes to the juju events being observed (newly added, significantly modified or deleted) -->
+
+### Module Changes
+
+<!-- Any high level changes to modules and why (Service, Observer, helper) -->
+
+### Library Changes
+
+<!-- Any changes to charm libraries -->
+
+### Checklist
+
+- [ ] The [charm style guide](https://juju.is/docs/sdk/styleguide) was applied
+- [ ] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
+- [ ] The changes are compliant with [ISD014 - Managing Charm Complexity](https://discourse.charmhub.io/t/specification-isd014-managing-charm-complexity/11619)
+
```

Comment:
I've seen other charm PR templates have more than 3 checks. Is this intended?

Reply:
@jdkandersson Should we use the following items in the template?

```
+ - [ ] The documentation is generated using `src-docs`
+ - [ ] The documentation for charmhub is updated.
+ - [ ] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)

Reply:
yes could you please add them to the spec as well?

Reply:
Changed. I just didn't change the spec because it might restart the approval process.